docs/pages/reference/backends.mdx

```diff
@@ -80,9 +80,13 @@ If your load balancer supports HTTP health checks, configure it to hit the
 `/readyz` [diagnostics endpoint](../management/diagnostics/monitoring.mdx) on
 machines running Teleport. This endpoint must be enabled by using the
 `--diag-addr` flag to teleport start: `teleport start
---diag-addr=127.0.0.1:3000` The
-[http://127.0.0.1:3000/readyz](http://127.0.0.1:3000/readyz) endpoint will
+--diag-addr=0.0.0.0:3000` The
+[http://0.0.0.0:3000/readyz](http://0.0.0.0:3000/readyz) endpoint will
```

Comment:
- 0.0.0.0 is a notation for listening, but you can't dial it, so I would just say "the /readyz endpoint"
- I don't know why we format this as a link. Clicking this link in the docs won't take the reader anywhere useful
- 
```suggestion
`/readyz` endpoint will
```